packages/block-editor/src/components/preview-options/README.md

```diff
@@ -0,0 +1,63 @@
+# Preview Options
+
+The `PreviewOptions` component displays the list of different preview options available in the editor.
+
+It returns a [`DropdownMenu`](https://github.com/WordPress/gutenberg/tree/master/packages/components/src/dropdown-menu) component with these different options. The options currently available in the editor are Desktop, Mobile, Tablet and "Preview in new tab".
+
+![Preview options dropdown menu](https://make.wordpress.org/core/files/2020/09/preview-options-dropdown-menu.png)
+
+## Table of contents
+
+1. [Development guidelines](#development-guidelines)
+2. [Related components](#related-components)
+
+## Development guidelines
+
+### Usage
+
+Renders the previews options of the editor in a dropdown menu.
+
+```jsx
+import { PreviewOptions } from '@wordpress/block-editor';
```

Comment:
This component is still being exported as `__experimental`; we should probably document its usage as `import { __experimentalPreviewOptions as PreviewOptions } from '@wordpress/block-editor';`. There is some precedent for this in other experimental component readmes, though we'll have to then remember to update the docs once it's moved out of experimental status 😅

Reply:
So, it's fine to document experimental components I guess.